### PR TITLE
[plugin] : align tree item description properly

### LINKED ISSUE #7199: [regression] description is not aligned properly in the plugin tree
It should be aligned to the right.

Now it is:
Screenshot 2020-02-24 at 11 26 24

How it used to be:
Screenshot 2020-02-24 at 11 27 16

@vince-fugnitto I am not sure which styling PR broke it. It would be good to find and revert it.

### PR DESCRIPTION
**Pull request by akosyakov**
#### What it does

- fix #7199

#### How to test

- install vscode tree view sample extension: https://github.com/microsoft/vscode-extension-samples/tree/master/tree-view-sample
- check the description is aligned to left
- check that ellipses are shown for the caption while resizing

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by akosyakov** [comment]
cc @vince-fugnitto feel free to merge if it is good for the release

File: packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx [hunk 1]
```diff
@@ -254,7 +254,7 @@ export class TreeViewWidget extends TreeWidget {
         return React.createElement('div', attrs, ...children);
     }
 
-    protected getCaption(node: TreeNode): React.ReactNode[] {
+    protected getCaption(node: TreeNode): React.ReactNode {
```
Comment (akosyakov) [inline packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx]: The main trick is that we turn each element to span that it does not create multiple lines and then wrap them into div element with block display and `noWrapInfo` class install. It makes sure that all span are treated as one element and get trimmed accordingly only once.

Comment (akosyakov) [reply packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx]: I had to solve this issue already many times and each time it is hard for me, had to remember that text overflow does not play nicely with flex display.

**Comment by vince-fugnitto** [comment]
I'm including the fix in the release since it fixes an issue that affects all plugin trees, and was introduced during the month.

**Pull request merged**